pkgs/top-level/all-packages.nix

```diff
@@ -22709,7 +22709,7 @@ in
 
   ### SCIENCE/ROBOTICS
 
-  apmplanner2 = libsForQt59.callPackage ../applications/science/robotics/apmplanner2 { };
+  apmplanner2 = libsForQt511.callPackage ../applications/science/robotics/apmplanner2 { };
```

Comment:
```suggestion
  apmplanner2 = libsForQt5.callPackage ../applications/science/robotics/apmplanner2 { };
```

Reply:
Nope. That's on purpose. Changing it according to your recommendation breaks the build for me:
```
build flags: -j16 -l16 SHELL=/nix/store/5p8qcz6i13ymgf80kimhx2g9pb5r5nia-bash-4.4-p23/bin/bash
/nix/store/ais74bm3fvrcj6gcx3mq2l1n51xw5gag-qtbase-5.12.0-dev/bin/uic libs/opmapcontrol/src/mapwidget/mapripform.ui -o build-release/ui/ui_mapripform.h
/nix/store/ais74bm3fvrcj6gcx3mq2l1n51xw5gag-qtbase-5.12.0-dev/bin/uic src/ui/MainWindow.ui -o build-release/ui/ui_MainWindow.h
/nix/store/ais74bm3fvrcj6gcx3mq2l1n51xw5gag-qtbase-5.12.0-dev/bin/uic src/ui/CommSettings.ui -o build-release/ui/ui_CommSettings.h
/nix/store/ais74bm3fvrcj6gcx3mq2l1n51xw5gag-qtbase-5.12.0-dev/bin/uic src/ui/SerialSettings.ui -o build-release/ui/ui_SerialSettings.h
/nix/store/ais74bm3fvrcj6gcx3mq2l1n51xw5gag-qtbase-5.12.0-dev/bin/uic src/ui/UASControl.ui -o build-release/ui/ui_UASControl.h
/nix/store/ais74bm3fvrcj6gcx3mq2l1n51xw5gag-qtbase-5.12.0-dev/bin/uic src/ui/UASList.ui -o build-release/ui/ui_UASList.h
/nix/store/ais74bm3fvrcj6gcx3mq2l1n51xw5gag-qtbase-5.12.0-dev/bin/uic src/ui/UASInfo.ui -o build-release/ui/ui_UASInfo.h
/nix/store/ais74bm3fvrcj6gcx3mq2l1n51xw5gag-qtbase-5.12.0-dev/bin/uic src/ui/UASView.ui -o build-release/ui/ui_UASView.h
/nix/store/ais74bm3fvrcj6gcx3mq2l1n51xw5gag-qtbase-5.12.0-dev/bin/uic src/ui/ParameterInterface.ui -o build-release/ui/ui_ParameterInterface.h
/nix/store/ais74bm3fvrcj6gcx3mq2l1n51xw5gag-qtbase-5.12.0-dev/bin/uic src/ui/WaypointList.ui -o build-release/ui/ui_WaypointList.h
/nix/store/ais74bm3fvrcj6gcx3mq2l1n51xw5gag-qtbase-5.12.0-dev/bin/uic src/ui/ObjectDetectionView.ui -o build-release/ui/ui_ObjectDetectionView.h
/nix/store/ais74bm3fvrcj6gcx3mq2l1n51xw5gag-qtbase-5.12.0-dev/bin/uic src/ui/JoystickWidget.ui -o build-release/ui/ui_JoystickWidget.h
/nix/store/ais74bm3fvrcj6gcx3mq2l1n51xw5gag-qtbase-5.12.0-dev/bin/uic src/ui/HDDisplay.ui -o build-release/ui/ui_HDDisplay.h
/nix/store/ais74bm3fvrcj6gcx3mq2l1n51xw5gag-qtbase-5.12.0-dev/bin/uic src/ui/MAVLinkSettingsWidget.ui -o build-release/ui/ui_MAVLinkSettingsWidget.h
/nix/store/ais74bm3fvrcj6gcx3mq2l1n51xw5gag-qtbase-5.12.0-dev/bin/uic src/ui/AudioOutputWidget.ui -o build-release/ui/ui_AudioOutputWidget.h
/nix/store/ais74bm3fvrcj6gcx3mq2l1n51xw5gag-qtbase-5.12.0-dev/bin/uic src/ui/QGCSensorSettingsWidget.ui -o build-release/ui/ui_QGCSensorSettingsWidget.h
/nix/store/ais74bm3fvrcj6gcx3mq2l1n51xw5gag-qtbase-5.12.0-dev/bin/uic src/ui/watchdog/WatchdogControl.ui -o build-release/ui/ui_WatchdogControl.h
/nix/store/ais74bm3fvrcj6gcx3mq2l1n51xw5gag-qtbase-5.12.0-dev/bin/uic src/ui/watchdog/WatchdogProcessView.ui -o build-release/ui/ui_WatchdogProcessView.h
/nix/store/ais74bm3fvrcj6gcx3mq2l1n51xw5gag-qtbase-5.12.0-dev/bin/uic src/ui/watchdog/WatchdogView.ui -o build-release/ui/ui_WatchdogView.h
/nix/store/ais74bm3fvrcj6gcx3mq2l1n51xw5gag-qtbase-5.12.0-dev/bin/uic src/ui/QGCFirmwareUpdate.ui -o build-release/ui/ui_QGCFirmwareUpdate.h
/nix/store/ais74bm3fvrcj6gcx3mq2l1n51xw5gag-qtbase-5.12.0-dev/bin/uic src/ui/QGCPxImuFirmwareUpdate.ui -o build-release/ui/ui_QGCPxImuFirmwareUpdate.h
make: *** No rule to make target 'QOSGWidget.h', needed by 'build-release/ui/ui_QMap3D.h'.  Stop.
make: *** Waiting for unfinished jobs....
builder for '/nix/store/0gzs757iykcxksiahi7kccig55bp59zi-apmplanner2-2.0.27-rc1.drv' failed with exit code 2
error: build of '/nix/store/0gzs757iykcxksiahi7kccig55bp59zi-apmplanner2-2.0.27-rc1.drv' failed
```

Reply:
Ok